tests/standalone_tests/test_cleanup_after_algorithm_execution.py

```diff
@@ -189,8 +248,30 @@ async def test_cleanup_after_uninterrupted_algorithm_execution(
             algorithm_name=algorithm_name,
             algorithm_request_dto=algorithm_request_dto,
             datasets_per_local_node={
-                localnode1_tasks_handler.node_id: ["edsd"],
-                localnode2_tasks_handler.node_id: ["edsd"],
+                localnode1_tasks_handler.node_id: [
+                    "edsd0",
+                    "edsd1",
+                    "edsd2",
+                    "edsd3",
+                    "edsd4",
+                    "edsd5",
+                    "edsd6",
+                    "edsd7",
+                    "edsd8",
+                    "edsd9",
+                ],
```

Comment:
these clutter the logic of the logic test

Reply:
Will be fixed with https://team-1617704806227.atlassian.net/browse/MIP-198